shared/chat/new-team-dialog-container.js

```diff
@@ -2,19 +2,23 @@
 import NewTeamDialog from '../teams/new-team'
 import {connect} from 'react-redux'
 import {compose, withState, withHandlers} from 'recompose'
+import {type TypedState} from '../constants/reducer'
 import {createNewTeamFromConversation} from '../actions/teams/creators'
-import {selectConversation} from '../actions/chat/creators'
+import {upperFirst} from 'lodash'
+
+const mapStateToProps = (state: TypedState) => ({
```

Comment:
I have a question for you all: this subscribes this component to all state changes, right? Is there a preferred method or does it not matter for a little dialog?

Reply:
@chrisnojima